index.md

```diff
@@ -21,11 +21,12 @@ title: Distributed SQL query engine for big data
   </div>
 </div>
 
-<div class="trino-birthday-banner">
-  <div class="trino-birthday-content">
-    <h1>Trino is celebrating its 10th Birthday!</h1>
-    <a class="btn trino-birthday-button" href="{{ site.url }}{% post_url 2022-08-08-trino-tenth-birthday %}">Learn more</a>
-    <img src="./assets/images/trino-birthday-banner-peek.png" class="trino-birthday-peek">
+<div class="trino-summit-banner">
+  <div class="trino-summit-content">
+    <img src="/assets/images/trino-summit-logo.png" class="trino-summit-logo" />
+    <h3>November 10th, 2022</h3>
+    <a class="btn trino-summit-button" href="https://www.starburst.io/info/trinosummit/" target="_blank">Register</a>
```

Comment:
This is good for now.. once we have the blog post about the summit from @bitsondatadev we should change the link to point to the blog post or add another button with a different text .. like Learn more